app/SymfonyRequirements.php

```diff
@@ -1,739 +0,0 @@
-<?php
```

Comment:
Die is uit composer gehaald. En is een opmerking van SensioInsights dat die file er niet mag staan.